actions/tamper.py

```diff
@@ -82,11 +120,19 @@ def __str__(self):
         """
         s = Action.__str__(self)
         if self.tamper_type == "corrupt":
-            s += "{%s:%s:%s}" % (self.tamper_proto_str, self.field, self.tamper_type)
-        elif self.tamper_type in ["replace", "add"]:
-            s += "{%s:%s:%s:%s}" % (self.tamper_proto_str, self.field, self.tamper_type, self.tamper_value)
+            if self.tamper_proto == HTTPRequest:
+                s += "{%s:%s:%s:%s}" % (self.tamper_proto_str, self.field, self.tamper_type, str(self.start_index) + "-" + str(self.end_index))
```

Comment:
i don't love how layer-aware this has to be to work - can it be cleaned up at all? if not, no biggie, but it'd be nicer if we could clean this up